### ISSUE TITLE
Playbooks template "Prompt-User_alert" bug

### ISSUE THREAD

**Issue by ityankel** [Azure/Azure-Sentinel#2780]
**Describe the bug**
The template fails to be deployed with an error message about invalid action syntax:

> {"code":"DeploymentFailed","message":"At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/DeployOperations for usage details.","details":[{"code":"InvalidTemplate","message":"The template validation failed: 'The template action 'Post_a_message_(V3)' at line '1' and column '2777' is not valid: \"Unable to parse template language expression '*sanitized*': expected token 'LeftParenthesis' and actual 'Identifier'.\".'."}]}

**To Reproduce**
Deploy the Prompt-User alert trigger playbook

**Screenshots**
![image](https://user-images.githubusercontent.com/47292630/128051134-7b0f1650-d301-4fa8-a20f-014f88667d99.png)

**Comment by dicolanl** [comment]
hi @ityankel 
I just tested deploy of the alert trigger playbook.  it worked fine.
![image](https://user-images.githubusercontent.com/25964057/128214083-ed94d0b0-e1c6-4486-a7d0-392497056311.png)

are you using azure commercial or gov?  i tested commercial

**Comment by ityankel** [comment]
@dicolanl , I checked it again and I think I understand the issue - trying to deploy the template with the default parameter values leads to the error I encountered when trying to reproduce the same error that a few customers using the playbooks gallery in the Sentinel portal encountered. Replacing them with a valid id allows the template to be deployed correctly.

Can we maybe remove the default value from the 2 teams parameters to make this mistake less likely?
